src/backends/perf_pt/perf_pt_private.h

```diff
@@ -47,6 +47,18 @@
 #include <inttypes.h>
 #include <stdbool.h>
 
+enum perf_pt_cerror_type {
+    perf_pt_cerror_unknown,
+    perf_pt_cerror_errno,
+    perf_pt_cerror_ipt,
+};
+
+struct perf_pt_cerror {
+    int typ;    // Is it a errno or libipt error code.
+    int code;   // The error code itself.
+    bool set;   // Used to ensure newer failures don't overwrite older ones.
```

Comment:
The `bool` seems pointless (and bloats the struct because of alignment!): if you have an entry in the enum `Unused` then it subsumes the bool AFAICS?

Reply:
Good idea.

Reply:
Fix pushed.